src/search/component/RealTimeKeyword/RealTimeKeyword.tsx

```diff
@@ -0,0 +1,127 @@
+import { useEffect, useState } from 'react';
+
+import { ListItem, IcSearchLine } from '@yourssu/design-system-react';
+import { useNavigate } from 'react-router-dom';
+
+import RealTimeKeywordImage from '@/assets/search/RealTimeKeyword.png';
+
+import {
+  StyledContainer,
+  StyledHeader,
+  StyledHeaderImageSection,
+  StyledHeaderText1,
+  StyledHeaderText2,
+  StyledHeaderTextSection,
+  StyledList,
+  StyledListItemRanking,
+  StyledListItemText,
+} from './RealTimeKeyword.style';
+
+interface RealTimeData {
+  year: string;
+  month: string;
+  day: string;
+  hour: string;
+  min: string;
+}
+
+// Keyword 가져오는 쿼리 작성
+export const RealTimeKeyword = () => {
+  const navigate = useNavigate();
+
+  const [realTime, setRealTime] = useState<RealTimeData>();
+
+  useEffect(() => {
+    const koreaTime = new Date().toLocaleString('en-US', { timeZone: 'Asia/Seoul' });
+    const [date, time] = koreaTime.split(', ');
+    const [month, day, year] = date.split('/');
+    const [hour, min, _] = time.split(':');
+    const [, AMPM] = _.split(' ');
+
+    const setMonth = month.length === 1 ? '0' + month : month;
+    const setDay = day.length === 1 ? '0' + day : day;
+    const setMin = min.length === 1 ? '0' + min : min;
```

Comment:
month, day, min을 두 자리수로 보여주기 위해 string.length를 사용하셨는데
string.padStart()를 쓰면 코드가 더 간결해질 거 같아요 ㅎㅅㅎ
문자열이 지정된 자리수보다 짧을 때 앞을 채워주는 함수입니다-!

```suggestion
    const setMonth = month.padStart(2, '0');
    const setDay = day.padStart(2, '0');
    const setMin = min.padStart(2, '0');
```